apps/postybirb-ui/src/components/form/fields/field.css

```diff
@@ -1,3 +1,7 @@
 .postybirb-field-error {
-  background-color: rgba(200, 0, 0, 0.2);
-}
\ No newline at end of file
+  background-color: rgba(200, 0, 0, 0.1);
+}
```

Comment:
changed alpha from 0.2 to 0.1 because it looked weird in light theme

this is how it looks now:
<img width="736" height="210" alt="Image" src="https://github.com/user-attachments/assets/7501ac94-50f3-49a7-b7d7-94f0b606ad02" />
<img width="708" height="209" alt="Image" src="https://github.com/user-attachments/assets/09f0ce23-8e88-4d64-8072-2742bc68cc02" />